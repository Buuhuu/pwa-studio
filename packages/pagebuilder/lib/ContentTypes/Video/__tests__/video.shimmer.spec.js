import React from 'react';

import { createTestInstance } from '@magento/peregrine';

import VideoShimmer from '../video.shimmer';

jest.mock('@magento/venia-ui/lib/classify');

test('renders an empty VideoShimmer component', () => {
    const component = createTestInstance(<VideoShimmer />);

    expect(component.toJSON()).toMatchSnapshot();
});

test('renders a VideoShimmer component with maxWidth', () => {
    const props = {
        maxWidth: '1200px'
    };
    const component = createTestInstance(<VideoShimmer {...props} />);

    expect(component.toJSON()).toMatchSnapshot();
});
