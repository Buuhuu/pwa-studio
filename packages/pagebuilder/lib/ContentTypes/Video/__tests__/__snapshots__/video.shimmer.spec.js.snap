// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a VideoShimmer component with maxWidth 1`] = `
<div
  aria-busy="true"
  aria-live="polite"
  className="root_rectangle"
  style={
    Object {
      "marginBottom": undefined,
      "marginLeft": undefined,
      "marginRight": undefined,
      "marginTop": undefined,
      "maxWidth": "1200px",
    }
  }
>
  <div
    className="inner"
  >
    <div
      className="wrapper"
      style={
        Object {
          "border": undefined,
          "borderWidth": undefined,
          "paddingBottom": undefined,
          "paddingLeft": undefined,
          "paddingRight": undefined,
          "paddingTop": undefined,
        }
      }
    >
      <div
        className="container"
      />
    </div>
  </div>
</div>
`;

exports[`renders an empty VideoShimmer component 1`] = `
<div
  aria-busy="true"
  aria-live="polite"
  className="root_rectangle"
  style={
    Object {
      "marginBottom": undefined,
      "marginLeft": undefined,
      "marginRight": undefined,
      "marginTop": undefined,
      "maxWidth": undefined,
    }
  }
>
  <div
    className="inner"
  >
    <div
      className="wrapper"
      style={
        Object {
          "border": undefined,
          "borderWidth": undefined,
          "paddingBottom": undefined,
          "paddingLeft": undefined,
          "paddingRight": undefined,
          "paddingTop": undefined,
        }
      }
    >
      <div
        className="container"
      />
    </div>
  </div>
</div>
`;
